scripts/pylint_extensions.py

```diff
@@ -757,6 +757,33 @@ def process_module(self, node):
                         'backslash-continuation', line=line_num + 1)
 
 
+class FunctionArgsOrderChecker(checkers.BaseChecker):
+    """Checker for order of arguments. It checks
+       that class arguments comes first.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'function-args-order'
+    priority = -1
+    msgs = {
+        'C0005': ('Wrong order of arguments '
+                  'self should come first',
+                  'wrong-ord-args-self',
```

Comment:
The `name` should be same everywhere. Therefore, change this to `function-args-order`.

Reply:
```name``` can't be same as there are two messages so for each message name should be different so that we can differentiate between them.

Reply:
[This](https://github.com/oppia/oppia/pull/5919/files/ac6943f6b6dd6e1a94432dc0c2e802c12010e628#diff-0aaddd24feb0c0ba5defe2b2fb6dccf7R766) is the name of the checker. There is no checker of the name `wrong-order-args-self`. Did you try running this checker once?

Reply:
Let's keep a common name and differentiate on the basis of the message.

Reply:
Actually 'wrong-ord-args-self' is the message-id, which should be unique.

Reply:
Yes, it is unique for a _particular_ checker. We can distinguish on the basis of the message in this case.

Reply:
Tested the checker by making a test.py file !
![screenshot from 2018-12-20 20-32-14](https://user-images.githubusercontent.com/30555544/50296238-0e238c80-04a0-11e9-923b-f811749b0437.png)